src/test/java/uk/gov/justice/probation/courtcaseservice/jpa/repository/HearingRepositoryFacadeIntTest.java

```diff
@@ -58,6 +58,20 @@ public void whenFindFirstByHearingIdOrderByIdDesc_thenReturnCorrectRecordWithOff
         assertIsFerrisBueller(actual);
     }
 
+    @Test
+    public void whenFindFirstByHearingIdAndCourtCaseId_thenReturnCorrectRecordWithOffender() {
+        final var actual = hearingRepositoryFacade.findFirstByHearingIdAndCourtCaseId("5564cbfd-3d53-4f36-9508-437416b08738", "727af2a3-f9ec-4544-b5ef-2ec3ec0fcf2b");
+
+        assertIsFerrisBueller(actual);
+    }
+
+    @Test
+    public void whenFindFirstByHearingIdAndCourtCaseId_thenDoNotReturnRecord() {
```

Comment:
I'm assuming `"827af2a3-f9ec-4544-b5ef-2ec3ec0fcf2b"` does not exist. 

is there an Integration test that has a hearing with a court case in the database and then we do a PUT /hearing with multiple cases (1 of them being the one that exists and the other doesn't)

testing the PUT